src/LandingPage/CreateUserLink.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import './LandingPage.css';
+//import { connect } from "react-redux";
+//import { Link } from 'react-router-dom';
+import Modal from 'react-responsive-modal';
+
+
+
+class CreateUserLink extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            open: false
+        };
+    };
+
+    // state = {
```

Comment:
Let's delete this commented out code, and the commented out code below here.

Reply:
done